HW18andmore/q1.txt

```diff
@@ -0,0 +1,25 @@
+INSERT INTO mynames
+    (first_name, last_name, age)
+VALUES ('Lili','L', 37)
+
+INSERT INTO mynames
+    (id, first_name, last_name)
```

Comment:
nit: I don't think it is a good idea to add an `id` value cause it is `AUTOINCREMENT` and is created by default

Reply:
agree, just wanted to try